source/administration/plugins.rst

```diff
@@ -272,7 +272,7 @@ Custom plugins
 ~~~~~~~~~~~~~~
 Installing a custom plugin introduces some risk. As a result, plugin uploads are disabled by default and cannot be enabled via the System Console or REST API.
 
-To enable plugin uploads, manually set **PluginSettings > EnableUploads** to ``true`` in your configuration and restart your server. You can disable plugin uploads at any time without affecting previously uploaded plugins.
+To enable plugin uploads, manually set **PluginSettings > EnableUploads** to ``true`` in your config.json file and restart your server. You can disable plugin uploads at any time without affecting previously uploaded plugins.
```

Comment:
```suggestion
To enable plugin uploads, manually set **PluginSettings > EnableUploads** to ``true`` in your ``config.json`` file and restart your server. You can disable plugin uploads at any time without affecting previously uploaded plugins.
```

Reply:
I think these steps refer to the System Console as the config.json file doesn't include the fields **PluginSettings > Enable Uploads**. The config.json reference [here](https://docs.mattermost.com/administration/config-settings.html#enable-plugins) differs slightly. 

That said, it looks like the instructions for the System Console need to be updated as there is no **Enable Plugins** option in **Plugin Management**. 

@aaronrothschild, can you confirm that the steps should be **System Console > Plugins (Beta) > Plugin Management > Enable Plugins** to **true**?

Reply:
There is the field in the config.json. See my screenshot when you go to the permlink.